.github/ISSUE_TEMPLATE/feature-enhancement-request.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Feature enhancement request
+about: Suggest an idea for ytt
+labels: carvel-triage
+---
+
+**Describe the problem/challenge you have**
+[A description of the current limitation/problem/challenge that you are experiencing.]
+
+
+**Describe the solution you'd like**
+[A clear and concise description of what you want to happen.]
```

Comment:
In the description, it might be nice to prompt the user to include a visual example of what the enhancement would look like. For example, if a user requested a way for users to skip hidden directories via an option, it would be great to see the proposed options used with `ytt` like below:

```
ytt -f file.yaml -f file2.yaml --skip-hidden-dir
```

Reply:
done